### PR TITLE
Editing the LANDING PAGE!

### PR DESCRIPTION
**Pull request by KP-TVu**
Team: I've made some changes to the repository landing page, please make suggestions for editing, or approve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# Michelle_Test
\ No newline at end of file
+# Michelle_Test
+
+This is the repository I setup for our little talk.
+
+Michelle is learning *PYTHON*
```
Comment (michelle313xo) [inline README.md:5]: Michelle is already a PRO at Python

Comment (KP-TVu) [reply README.md:5]: Okay! I will change that to show your PRO status!

**Comment by michelle313xo** [comment]
This looks good, just make sure it runs

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+One of the initial code pieces is something that gets input from the user, and spits it back out!
+
+Here's an example.
+
+```
+first_name = input("What is your first name? ")
```
[the remark this answers is not in the document]
Comment (michelle313xo) [reply README.md:12]: The code works!

**Review by michelle313xo — APPROVED** [review]

**Pull request merged**